.circleci/config.yml

```diff
@@ -11,7 +11,7 @@ jobs:
           keys:
             - v3-dependencies-{{ checksum "build.gradle.kts" }}
             - v3-dependencies-
-      - run: ./gradlew compileTestKotlin
+      - run: ./gradlew compileTestKotlin --stacktrace
```

Comment:
> Cause 9: org.gradle.internal.locking.LockOutOfDateException: Did not resolve 'commons-logging:commons-logging:1.2' which is part of the dependency lock state

Some Gradle releases  change the lockfile format (major only maybe?).
I had to [nuke old ones in the past](https://github.com/atlassian/virtual-users/pull/50/files/ecfbc42547f5bc7afe8fd6597c3ed0a919f4c193..97e2c00c5d0b4639180a8e90d77e9cf37d1595a1). Maybe we gotta do it again?